triage/render.js

```diff
@@ -320,7 +320,19 @@ function renderCluster(top, cluster) {
     let [title, body] = makeGitHubIssue(id, text, owner, latestBuilds);
     title = encodeURIComponent(title);
     body = encodeURIComponent(body);
-    fileBug.href = `https://github.com/kubernetes/kubernetes/issues/new?body=${body}&title=${title}`;
+
+    // Open issue in kubernetes-sig/cluster-api repo for cluster api related CI issues.
+    let jobParams = (new URL(document.location)).searchParams;
+    let jobName = jobParams.get("job");
+
+    switch (jobName) {
+      case "periodic-cluster-api-e2e-main":
```

Comment:
I think we cannot hard-code a job name or a specific job name format here.

Is there any other way to figure out to which repository an issue belongs?

Reply:
For filtering, we have "Failure text", "Job", "Test". I think Job is the best filter we have. Should I make it so that if Job has the text "cluster-api" then we proceed? What do you think?

Reply:
I think matching for cluster-api is slightly better then matching for an exact job name, but it still feels pretty hacky to me.

I think ultimately we should ask the maintainers of this tool what they think and if they see better options

Reply:
You _might_ be able to figure out some metadata about which repo the job is created for (although this might be more tricky for periodic jobs) and then construct the URL as `https://github.com/<org>/<project name>/issues/new?body=${body}&title=${title}` which would be generic and solve this for all projects, not just CAPI

Reply:
https://github.com/kubernetes/test-infra/blob/master/triage/README.md has a bit of info on what data is available, potentially `executor` or `pr` looks interesting, or `job_paths`? it's hard to tell from the doc what info they each contain

Reply:
Yes, it's just giving the data type rather than having a concrete example.